template/src/ontology/run.sh.jinja2

```diff
@@ -37,15 +41,23 @@ fi
 VOLUME_BIND=$PWD/../../:/work
 WORK_DIR=/work/src/ontology
 
+if [ -n "$ODK_BINDS" ]; then
+    VOLUME_BIND="$VOLUME_BIND,$ODK_BINDS"
+fi
+
 if [ -n "$USE_SINGULARITY" ]; then
     {% if project.use_external_date is sameas True %}export TODAY=$(date +%Y-%m-%d){% endif %}
-    singularity exec --cleanenv \
+    singularity exec --cleanenv $ODK_SINGULARITY_OPTIONS \
         --env "ROBOT_JAVA_ARGS=$ODK_JAVA_OPTS,JAVA_OPTS=$ODK_JAVA_OPTS" \
         --bind $VOLUME_BIND \
```

Comment:
Should this receive the same love as the standard docker thing below? (in case ODK_BINDS is set)?

Reply:
No need. Contrary to Docker’s `-v` option, Singularity’s `--bind` option accepts multiple binding specifications in a single argument (provided they’re comma-separated), so passing `--bind $VOLUME_BIND` is enough. There is no need to split `$VOLUME_BIND` into multiple `--bind` options as is the case for Docker.

Reply:
Thanks